README.md

```diff
@@ -1 +1,48 @@
-# secure-api-gateway-ci  [![Nightly Build](https://github.com/SecureApiGateway/secure-api-gateway-ci/actions/workflows/nightly-build.yml/badge.svg)](https://github.com/SecureApiGateway/secure-api-gateway-ci/actions/workflows/nightly-build.yml)
\ No newline at end of file
+# secure-api-gateway-ci  
+NOTE:This repository is for internal PingID use only, and not designed for customers to base their deployments on.
+## Nightly Builds 
+[![Nightly Build](https://github.com/SecureApiGateway/secure-api-gateway-ci/actions/workflows/nightly-build.yml/badge.svg)](https://github.com/SecureApiGateway/secure-api-gateway-ci/actions/workflows/nightly-build.yml)
+
+This action will trigger each morning before the cluster is deployed. 
+
+Its purpose is to build each individual component from new (uses the -U option in the maven build command).
```

Comment:
Just a note, we want to always use the -U option with maven when doing CI builds. This will ensure that we are always building against the latest version of any snapshot dependencies.

Similarly, with gradle we always want to supply `--refresh-dependencies` to achieve the same thing.

Might be worth reviewing if this is the case.

Reply:
I think this is the case, as within the nightly build I'm just using the existing Merge commands, which use the Makefiles. Just wanted to document in here that they are being made from new, I'll reword it to say that its using the 'usual' makefile commands, which include the -u and --refresh-dependencies